### DISCUSSION TITLE
issue: field array status, once dirty stays dirty

### DISCUSSION THREAD

**Discussion by CrocoDillon** [react-hook-form/react-hook-form#10214] [Q&A]
### Version Number

7.37.0

### Codesandbox/Expo snack

https://codesandbox.io/s/react-hook-form-usefieldarray-forked-kkgvjt?file=/src/index.tsx

### Steps to reproduce

1. Click append
2. Remove appended item

### Expected behaviour

Field array's `isDirty` status should reset to `false`. All relevant (deep) fields in `dirtyFields` reset to `false` but they are not removed properly, so `isDirty` for the whole field array remains `true` even though nothing is changed.

Note that the form's `isDirty` status properly resets to `false`.

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by LoicKairon** [comment]
@CrocoDillon was there any update on this issue?

**Comment by thomas-cruz** [comment]
Any updates?

**Comment by CrocoDillon** [comment]
@LoicKairon @thomas-cruz I'm sorry I haven't checked this for a while. As far as I know the issue is still there.
